src/System.Reflection/tests/EventInfo/EventInfo_PropertyTests.cs

```diff
@@ -1,294 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Xunit;
-using System;
-using System.Reflection;
-using System.Collections.Generic;
-
-#pragma warning disable 0067
-
-namespace System.Reflection.Tests
-{
-    public class EventInfoPropertyTests
-    {
-        // Verify AddMethod for EventInfo
-        [Fact]
-        public void VerifyAddMethod1()
```

Comment:
The tests for the AddMethod, RemoveMethod and RaiseMethod properties are now combined into the tests for GetAddMethod(), GetRemoveMethod() and GetRaiseMethod() respectively